src/ui/public/typeahead/partials/typeahead-items.html

```diff
@@ -5,7 +5,10 @@
   class="typeahead-items"
   >
 
+  <div ng-if="!typeahead.query.length" ng-transclude></div>
```

Comment:
If you remove this ng-if and put the transclude after the suggestions, then users can access the help even when the typeahead is offering suggestions. We might need to figure out a way to let the use collapse/expand the help. I think we could do that in a subsequent PR.

![image](https://cloud.githubusercontent.com/assets/1238659/24471861/57173bf4-1478-11e7-94a7-dda9980ba1d4.png)


```html
<div
  ng-show="typeahead.isVisible()"
  ng-mouseenter="typeahead.setMouseover(true);"
  ng-mouseleave="typeahead.setMouseover(false);"
  class="typeahead-items"
  >

  <div
    ng-if="typeahead.query.length"
    ng-repeat="item in typeahead.getItems()"
    ng-class="{active: item === typeahead.active}"
    ng-click="typeahead.selectItem(item, $event);"
    ng-mouseenter="typeahead.activateItem(item);"
    class="typeahead-item"
    >
    {{item}}
  </div>

  <div ng-transclude></div>
</div>
```